extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -130,8 +131,24 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
       'amp-subscriptions-dialog'
     );
     ampSubscriptionsEl.addEventListener('click', (event) =>
-      this.onSkipButtonClick_(event)
+      this.onSubscriptionsDialogClick_(event)
```

Comment:
Should this function name change be reverted?

Reply:
I actually think this is better because it reflects better on that we are monitoring clicks on 'amp-subscriptions-dialog' instead of a specific button.